src/Evolution/Systems/GeneralizedHarmonic/Equations.hpp

```diff
@@ -127,4 +144,167 @@ struct ComputeNormalDotFluxes {
       const tnsr::II<DataVector, Dim>& inverse_spatial_metric,
       const tnsr::i<DataVector, Dim>& unit_normal) noexcept;
 };
+
+/*!
+ * \ingroup NumericalFluxesGroup
+ * \brief Computes the generalized-harmonic upwind flux
+ *
+ * The upwind flux in general is given by Eq. (6.3) of \cite Teukolsky2015ega :
+ * \f{eqnarray}{
+ * F^* = S \Lambda^+ S^{-1} u^- + S \Lambda^- S^{-1} u^+,
+ * \f}
+ * where \f$u\f$ is a vector of the evolved variables, \f$S^{-1}\f$
+ * maps the evolved variables into the characteristic variables \f$S^{-1}u\f$,
+ * \f$S\f$ maps the characteristic variables into the evolved variables,
+ * and \f$\Lambda\f$ is a diagonal matrix of the average characteristic
+ * speed at the interface.
+ *
+ * Here, \f$S^{-1}u^-\f$ represents the
+ * characteristic variables at the element interface, computed using
+ * evolved variables from the element interior;
+ * \f$S^{-1}u^+\f$ represents the characteristic variables at the element
```

Comment:
There are really two cases: int/ext for the fields and neg/pos for the sign of the speeds. I will use these instead of +/- in my next push to this PR, to avoid confusion with the individual fields U^+ and U^-.